### DISCUSSION TITLE
Image generation suddenly slower

### DISCUSSION THREAD

**Discussion by ludovitkapusta** [lllyasviel/Fooocus#1261] [Q&A]
Hey,
yesterday, image generations was running at 3s/it ... today it suddenly changed to 10s/it ... making image generation significantly slower without any reasonable explanation... only thing i did was restarting computer.

does anyone have any advice what could go wrong?

**Comment by jclevet** [comment]
May be an other application using GPU ? Or if you are on laptop, the power cable got unplugged?

**Reply by ludovitkapusta** [reply]
nah, its a desktop

another weird thing is that although GPU is going on 100% (temps are normal), i dont hear kicking fans like usual... maybe my pc is going to die :D

**Comment by ludovitkapusta** [comment]
maybe it will help someone
it seems that problem was at disk for swap file. it was too low on storage and swap almost didnt work

soclearing up storage or move disk for swap file should help

**Comment by ludovitkapusta** [comment]
so, it is doing again... one pc restart and its slow again.... so reconfigure of swap fle didnt helped as i thought
